### DISCUSSION TITLE
Host: Webpack + Remote: Vite

### DISCUSSION THREAD

**Discussion by rodoabad** [originjs/vite-plugin-federation#342] [Q&A]
Hello, people! Thanks for all the hard work you've done!

I have a question.

Has anyone ever tried to have a Webpack host for module federation and then a federated Vite remote?

Thanks!

**Comment by Jiannan-dev** [comment]
Hello , Here's a project with webpack and vite as host and remote for each other
https://github.com/originjs/vite-plugin-federation/tree/main/packages/examples/vue3-demo-webpack-esm-esm
